### DISCUSSION TITLE
Hosting 100% locally with no ties to AWS products?

### DISCUSSION THREAD

**Discussion by uhlhosting** [chaskiq/chaskiq#714] [Q&A]
Hi, 

any plans to add support for local storage and to allow SMTP gateways outside SES, not everyone enjoy AWS products, nor even want to have to deal with them, its a pity when such great project are tied to cloud platforms, what about we sysops and devops who have own solutions based on kubernetes, proxmox, postal and such?

I believe this has potential to increase adoption and overall community around the app.

**Comment by michelson** [comment]
you can do all of that, just use SMTP credentials for email, minio or local disk for storage. in terms of deployment just use any docker compatible platform

**Reply by uhlhosting** [reply]
@michelson thanks for reply, will run a testing spin on caprover.

**Reply by kalebwd** [reply]
@michelson Where are SMTP credentials configured? If in the environment variables, what are the needed variables?

**Reply by michelson** [reply]
SMTP_ADDRESS=
SMTP_PASSWORD=
SMTP_USERNAME=
SMTP_DELIVERY_METHOD=smtp
